pkg/configuration/configuration.go

```diff
@@ -38,6 +38,12 @@ const (
 
 // Configuration is used to configure the approval process for PRs in a given repository.
 type Configuration struct {
+	// DisableContributorReview will enforce that an reviewer can not approve a pull request that they have contributed
+	//towards. That is, an approvers review is only considered if they have _not_ pushed a commit to the branch being merged.
+	// This does not include UI merges from the repositories main branch.
+ 	// See https://github.com/form3tech-oss/github-team-approver/pull/27 for more details
+	// Defaults to false.
+	DisableContributorReview bool
```

Comment:
would it make sense to include `approve` as a verb in the variable name ? 
review on it's own I think won't be disabled, if I understand the functionality correctly, the difference is in whether the reviewer can approve or not ?

Reply:
I've updated to be a bit more clear. As you say a user can still review a PR and from Github's perspective they can actually still approve it, even users who are not code owners. From the view of Github Team Approver, when this config is enabled, the approval (or positive review) is ignored.